vispy/visuals/gridmesh.py

```diff
@@ -0,0 +1,95 @@
+
+from .mesh import MeshVisual
+from ..geometry import create_grid_mesh, MeshData
+
+
+class GridMeshVisual(MeshVisual):
+    """Displays a mesh in a Cartesian grid about x,y,z coordinates.
+
+    This makes it simple to generate a mesh from e.g. the output
+    of numpy.meshgrid.
+
+    All arguments are optional, though they can be changed
+    individually later with the set_data method.
+
+    Parameters
+    ----------
+    xs : ndarray
+        A 2d array of x coordinates for the vertices of the mesh. Must
+        have the same dimensions as ys and zs.
+    ys : ndarray
+        A 2d array of y coordinates for the vertices of the mesh. Must
+        have the same dimensions as xs and zs.
+    zs : ndarray
+        A 2d array of z coordinates for the vertices of the mesh. Must
+        have the same dimensions as xs and ys.
+    colors : ndarray
+        The colors of the points of the mesh. Should be either a
+        (width, height, 4) array of rgba colors at each grid point or
+        a (width, height, 3) array of rgb colors at each grid point.
+        Defaults to None, in which case 
```

Comment:
`None` should be in the `colors : ` line and this description should be finished